### DISCUSSION TITLE
TinaCMS with supabase or firebase

### DISCUSSION THREAD

**Discussion by kunjee17** [tinacms/tinacms#2436] [Help] [answered]
Hi, 

First of thanks for nice CMS. I really liked it. My first thought was it is only supporting github check ins from cloud. But then I found out plugin with strapi. And nice intro video created by @perkinsjr . 

So, I was curious is it possible to connect any api with TineCMS. I couldn't find hooks for saving data in docs. I might be missing something. It would be great if someone can point me to right direction. Basically I have small digital cards application backed by Supabase. I am thinking of having tina cms there as default editor.

**Comment by jeffsee55** [comment, marked as answer]
Tina has been through a few changes, it was initially a very broad toolkit which required more work to be successful. During that time, you could wire it up to work with any backend. While that's still _technically_ true, we're focused on making Tina work really well with Git-backed content. So having said that, you may have luck referring to the [legacy docs](https://pre-beta.tina.io/docs/reference/toolkit/cms/). 

We hope that the Git-backed solution can be extended to work with any data source in the future, but it's not a focus at the moment.

**Reply by kunjee17** [reply]
Thanks @jeffsee55 for answer. Got your point. And again thanks for nice CMS. There is little learning curve but I really like what you people are doing.
